src/Console/Command/TaskCommand.php

```diff
@@ -36,12 +36,12 @@ public function __construct(
         $this->setDescription($taskAttribute->description);
         $this->setAliases($taskAttribute->aliases);
 
-        $commandName = $taskAttribute->name;
+        $taskName = $taskAttribute->name;
```

Comment:
Note for myself (and the next review): I had a long conversation with the other me, and I think this change is OK even if I'm not convinced.